docs/root/api-v1/cluster_manager/cluster_hc.rst

```diff
@@ -37,6 +38,10 @@ timeout_ms
 interval_ms
   *(required, integer)* The interval between health checks in milliseconds.
 
+no_traffic_interval_ms
```

Comment:
Do we need v1 config? We're really trying to feature freeze, since this will be removed completely in the 1.8 cycle (1.6 is released in 1-2 weeks).

Reply:
OK, I will remove it. 

I looked at a set of tests that possibly consumes this config (think should just migrate those to v2), hence thinking to have it here.